doc/sphinx/Makefile

```diff
@@ -41,6 +41,13 @@ copy-docs: clean-copy-docs
 	cp  ../release/quickReference.pdf     source/language/quickReference.pdf
 	cp  ../release/chapelLanguageSpec.pdf source/language/chapelLanguageSpec.pdf
 
+primers: clean-primers
+	@echo
+	@echo "Generating primers from "'$$CHPL_HOME'"/test/release/examples to "'$$CHPL_HOME'"/doc/sphinx/source/primers"
+	./chpl2rst.py ../../test/release/examples/primers/*.chpl --output=rst --prefix=source/primers
```

Comment:
This Makefile shouldn't be in the release tarball any time soon. I'll add a comment so we don't forget.